test/release/examples/primers/domains.chpl

```diff
@@ -30,49 +30,46 @@
 config var n = 10;
 
 var RD: domain(3) = {1..n, 1..n, 1..n};
-writeln(RD);
+writeln(RD); // {1..10, 1..10, 1..10}
 
 //
 // Rectangular domains have a set of methods that enable convenient
 // reuse of existing domains.
 //
 // The :proc:`~ChapelDomain.expand` method returns a new domain that is expanded or
-// contracted depending on the sign of the offset argument.
+// contracted depending on the sign of the offset argument.  The
+// expansion/contraction happens symmetrically to the lower and upper bound.
 //
-var RDbigger = RD.expand((1,1,1));
-writeln(RDbigger);
-var RDsmaller = RD.expand((-1,-1,-1));
-writeln(RDsmaller);
+var RDexp = RD.expand((1,2,-2));
+writeln(RDexp); // {0..11, -1..12, 3..8}
 
 //
-// The :proc:`~ChapelDomain.exterior` method returns a new domain that is the
-// exterior portion of the current domain.  A positive offset specifies that
-// the exterior should be taken from the high bound; a negative offset, the low
-// bound.
+// The :proc:`~ChapelDomain.exterior` method returns a new domain that is an
+// exterior portion of the current domain.  This is not the "border" of the
+// domain (which is not a rectangular domain), in particular the new domain
+// will share no indices with the original domain.  Inputs specify the width
```

Comment:
> will share no indices with the original domain

I suggest appending this clarification: unless the offset is zero.

If so, I suggest splitting the sentence in two, starting the second sentence with "In particular".

Reply:
Good catch, I've added an extra example to show what happens in that case.